hgs-keyvault/azuredeploy.parameters.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+            "value": "uday"
+        },
+        "adminPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "imageSKU": {
+            "value": "2016-Datacenter"
+        },
+      
+        "dnsNameforLBIP": {
+            "value": "GEN-UNIQUE-13"
+        },
+		
+    "vaultName": {
```

Comment:
it seems the indenting is inconsistent in this file; could we run this through a json formatter?